src/toil/worker.py

```diff
@@ -209,6 +211,9 @@ def badWorker():
         logger.debug("Next available file descriptor: {}".format(
             nextOpenDescriptor()))
 
+        logger.info("Running Toil version %s.", version)
```

Comment:
@hannes-ucsc @natanlao Since only the _last_ 40kb of the worker logs are preserved by default we could potentially lose this information. I know the issue said to log these at the beginning of the run but logging at the end may be better - not a big deal though.